Client/src/main/java/edu/sdccd/cisc191/template/Client.java

```diff
@@ -32,7 +32,7 @@ public class Client extends Application {
     /**
      * PlantManagement object
      */
-    private PlantManagement manager;
+    private final PlantManagement manager;
```

Comment:
Changed the access modifier to final to prevent the inadvertent reuse of the manager object.